package.json

```diff
@@ -25,7 +25,7 @@
     "link-aa-win": "node ./scripts/link-bin.js packages/amplify-app/bin/amplify-app amplify-app-dev",
     "link-dev": "cd packages/amplify-cli && ln -s \"$(pwd)/bin/amplify\" \"$(yarn global bin)/amplify-dev\" && cd -",
     "link-win": "node ./scripts/link-bin.js packages/amplify-cli/bin/amplify amplify-dev",
-    "lint-check": "yarn eslint . --ext .js,.jsx,.ts,.tsx --max-warnings=846",
+    "lint-check": "yarn eslint . --ext .js,.jsx,.ts,.tsx --max-warnings=847",
```

Comment:
we must not increase this threshold. we should find a file that introduced a warning and fix it.
(or fix other in exchange, but fixing new code is much more preferable).

Reply:
fixed